0xProject/0x-monorepo
# [WIP] Replace lodash with built-ins where possible to reduce bundle size

## Description

* Replaces `_.isUndefined` and `_.isNull` with the built-in equivalents. 
* Adds a tslint rule and auto-fix going forward

[_.isUndefined(obj)](https://lodash.com/docs/4.17.11#isUndefined) replaced with `obj === undefined`. This is the same as the lodash implementation [source](https://github.com/lodash/lodash/blob/4.17.11/lodash.js#L12199-L12201)

[_.isNull(obj)](https://lodash.com/docs/4.17.11#isNull) replaced with `obj === null`. Also same as the lodash implementation [source](https://github.com/lodash/lodash/blob/4.17.11/lodash.js#L11950-L11952)

## Testing instructions

You can run `cd packages/tslint-config && yarn build && yarn test` to test the auto-fixer
Double-check that monorepo has no instances of the replaced methods
All tests pass on CI

## Types of changes





 * New feature (non-breaking change which adds functionality) 



## Checklist:
